content/en/service_management/workflows/datastore.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Apps Datastore
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1gVVWLg5P3EXkVXBc4qZtqKp_nqykDsFXCMmh6pbvxx0/edit" btn_hidden="false" header="Try the Preview!">}}
+Datastore is in Preview. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+The Apps Datastore offers a scalable, structured data storage solution within Datadog's App Builder and Workflow Automation products. It supports CRUD (Create, Read, Update, and Delete) operations and integrates seamlessly with Datadog's ecosystem to optimize persistent data storage without the need for external databases.
+
+You can interact with a datastore using an app or a workflow, or you can use the UI in the Datadog App.
+
+
+## Prerequisites
+
+To interact with the Apps Datastore, your Datadog account must have the following [permissions][6], all of which are included in the Datadog Standard Role:
```

Comment:
To clarify
* if you need to access the datastore page, you need all permissions
* if you only need to use datastore from app builder and/or workflow, you don't need `apps_datastore_manage` permission

Reply:
Ok, thanks! I will update